### ISSUE TITLE
Response.InputStreamBody's toString method closes input stream.

### ISSUE THREAD

**Issue by TJReinert** [OpenFeign/feign#1208]
The toString() on Response.InputStreamBody causes the underlying inputstream to be closed. 
https://github.com/OpenFeign/feign/commit/0a0ff3f192c3ae8325b6421763e92cb9343c5334

When using IntellJ the default behavior while debugging is to call any toString methods that have been overridden. 
![image](https://user-images.githubusercontent.com/25063471/78837826-888c5e80-79ba-11ea-8d06-793f81ebdef0.png)

You can disable this functionality but it would be nice if the toString method just returned a copy of the data. 

I have opened a ticket with IntelliJ to allow an 'excludes' list. 
https://youtrack.jetbrains.com/issue/IDEA-237009

Thoughts?

**Comment by SwapneelA** [comment]
Hi,
I was facing IO exception with message stream is closed . I disabled toString object view which you suggested above, I was able to read body from response. 
Thanks for sharing the information.

**Comment by kdavisk6** [comment]
I agree we should change this.  My suggestion would be to not ready the data in `toString()` at all.  Thoughts?
